forPublication/vsm.md

```diff
@@ -1,42 +1,36 @@
+You've tried to deliver smaller increments, but it is still a struggle to complete these in a reasonable amount of time. 
+One of the reasons might be that wasteful activities and unnecessary handovers are hiding in your development process. 
 #### A Value Stream Map (VSM) can help you uncover this kinds of waste in your software delivery process
-
-Value Stream Mapping is a form of lean-management and was one of the foundations that made the Toyota Production 
-System. It maps the series of events that take a product or service from the beginning of the specific process (inception) 
-until it reaches the customer. 
-<br/><br/> Even though it stems from a manufacturing world, it can be applied to software as well.
-There it is an extremely helpful technique because of the high cost of of context switching in 
-knowledge work and software development specifically (link to blog the hidden cost of context switching) 
- 
+Value Stream Mapping is a form of lean-management and was one of the foundations that made the Toyota Production System.
+It maps the series of events that take a product or service from the beginning of the specific process (inception) until it reaches the customer. 
+<br/><br/>
+Even though it stems from a manufacturing world, it can be applied to software as well.
+There it is an extremely helpful technique because of the high cost of of context switching in knowledge work and software development specifically.
+(link to blog the hidden cost of context switching) 
 #### How to do it for a software project?
-I recommend doing it in a workshop format where everyone who is involved in the value stream is gathered 
-(from inception to delivery of the feature to the customers). The technique is simple (but the devil is in the details).
+I recommend doing it in a workshop format where everyone who is involved in the value stream is gathered (from inception to delivery of the feature to the customers).
+The technique is simple (but the devil is in the details).
 Choose a few relevant and representative example slices to map the value stream horizontally from left to right. 
-Now vertically map the finer grained tasks including wait times and possible loops between them.<br/><br/>
-Next, classify each fine-grained task as Customer Value Adding (CVA), Non Value Adding(NVA) and Waste. This is harder 
-then it seems and expect a lot of healthy discussion. 
+Now vertically map the finer grained tasks including wait times and possible loops between them.
+<br/><br/>
+Next, classify each fine-grained task as Customer Value Adding (CVA), Non Value Adding(NVA) and Waste.
+This is harder then it seems and expect a lot of healthy discussion. 
 <br/><br/>
-Add the average processing and wait times for the item. It is now possible to calculate cycle efficiency! Do not
-be surprised if your cycle efficiency is below 30%! 
+Add the average processing and wait times for the item.
+It is now possible to calculate cycle efficiency!
```

Comment:
If you want to protect yourself from VSM purists, you might want to say something like that you can calculate something approaching the cycle time/efficiency.
I think how we use this tool until now for software development is a simplified version, because we don't take into account parallel resource availability and loopbacks and stuff, which also doesn't make a lot of sense because we're working with averages anyway.